docs/gke-installation.md

```diff
@@ -0,0 +1,127 @@
+# Deploying Antrea on a GKE cluster
+
+We support running Antrea inside of GKE clusters on Ubuntu host. Antrea would operate
+in noEncap mode, in which no encapsulation is required for any kind of traffic (Intra Node,
+Inter Node etc) and NetworkPolicies are enforced using OVS.
+
+## GKE Prerequisites
+
+1. Install the Google Cloud SDK (gcloud). Refer to this (https://cloud.google.com/sdk/install)
```

Comment:
Consider: Refer to [Google Cloud SDK installation guide](https://cloud.google.com/sdk/install).

Reply:
Done